channeldb/migrations.go

```diff
@@ -300,3 +300,72 @@ func migrateInvoiceTimeSeriesOutgoingPayments(tx *bolt.Tx) error {
 
 	return nil
 }
+
+// migrateEdgePolicies is a migration function that will update the edges
+// bucket. It ensure that edges with unknown policies will also have an entry
+// in the bucket. After the migration, there will be two edge entries for
+// every channel, regardless of whether the policies are known.
+func migrateEdgePolicies(tx *bolt.Tx) error {
+	nodes := tx.Bucket(nodeBucket)
+	if nodes == nil {
+		return ErrGraphNotFound
```

Comment:
I think we can safely return `nil` if the buckets are not found? We are probably then trying to migrate an empty DB.

Reply:
Yes, indeed.